src/main/java/org/jenkinsci/plugins/ghprb/jobdsl/GhprbPullRequestMergeContext.java

```diff
@@ -93,4 +96,18 @@ public void allowMergeWithoutTriggerPhrase(boolean allowMergeWithoutTriggerPhras
     public void allowMergeWithoutTriggerPhrase() {
         allowMergeWithoutTriggerPhrase(false);
     }
+
+    /**
+     * @param onlyApprovedCode Allows only approved changes to be merged. Defaults to {@code false}.
+     */
+    public void onlyApprovedCode(boolean onlyApprovedCode) {
+        this.onlyApprovedCode = onlyApprovedCode;
+    }
+
+    /**
+     * Allows only approved changes to be merged. Defaults to {@code false}.
+     */
+    public void onlyApprovedCode() {
+        onlyApprovedCode(true);
```

Comment:
should this be false here?

Reply:
I actually tried out true and false here, I don't see that this has any effect. I added this for consistency because the other settings also have this kind of helper method